app/js/components/tabs/tabcomponents/savedComponent.jsx

```diff
@@ -253,11 +247,6 @@ class SavedComponent extends React.Component {
     }
   }
 
-  /**
-   * Method to search for saved definition queries and update the state
-   * @param {Object} event - Object containing details of this event
-   * @return {undefined}
-   */
   searchSavedDefinitions(event) {
```

Comment:
@Charpell Thank you.

Regarding your comment on the removal of the JSDOCs; I believe one of the best practices in coding is to make code readable, rather than always use comments. You will therefore notice that I have only left the jsdocs where they play a vital role in describing in depth, what a function does. 

However, if the function is clear about what it does, starting from the naming, then the JSDOC just ends up being redundant and fills up the file, making it look tardy. 
Case in point: on these lines you pointed out.  What role was the JSDOC really playing? It just repeats what the title already tells, and also misinforms one that the function should return an undefined value.

Commenting is important, but when overdone, it makes the codebase messy, can be misleading and just adds bulkiness to files. Here is a resource that addresses some of the best practices to follow when commenting code https://code.tutsplus.com/tutorials/top-15-best-practices-for-writing-super-readable-code--net-8118. I'll particularly call your attention to point Number 3.  Avoid Obvious Comments  `Commenting your code is fantastic; however, it can be overdone or just be plain redundant.`

I hope this adequately addresses your concern.  Feel free to reach out with any other concerns or observation. I appreciate your feedback.

Reply:
@LWanjiru Thank you for writing. I appreciate your feedback and your perspective response. Your points have been noted for future endeavours

Reply:
To add to the above, many developers change the code and forget to update the comments, resulting into the two being out of sync. That is why self documenting code will always win. 😊